tensorflow/python/kernel_tests/rnn_cell_test.py

```diff
@@ -105,6 +105,53 @@ def testBasicLSTMCell(self):
                                     m.name: 0.1 * np.ones([1, 4])})
         self.assertEqual(len(res), 2)
 
+  def testLSTMCellBlock(self):
+    with self.test_session() as sess:
+      with tf.variable_scope("root", initializer=tf.constant_initializer(0.5)):
+        x = tf.zeros([1, 2])
+        m = tf.zeros([1, 8 * 7])
+        g, out_m = tf.nn.rnn_cell.MultiRNNCell(
```

Comment:
Can I convince you to test against the BasicLSTMCell for both fw and backprop w.r.t. inputs & variables?